File: backend/src/main/java/com/backend/api/question/dto/response/PortfolioListReadResponse.java
package com.backend.api.question.dto.response;

import java.util.List;


public record PortfolioListReadResponse(
        String title,
        Long count,
        List<PortfolioReadResponse> questions
) {
    public static PortfolioListReadResponse from(String title,Long count,List<PortfolioReadResponse> questions) {
        return new PortfolioListReadResponse(title,count, questions);
    }
}

File: backend/src/main/java/com/backend/api/question/dto/response/PortfolioListReadResponse.kt
package com.backend.api.question.dto.response


data class PortfolioListReadResponse(
    val title: String,
    val count: Long,
    val questions: List<PortfolioReadResponse>
) {
    companion object {
        fun from(
            title: String,
            count: Long,
            questions: List<PortfolioReadResponse>
        ): PortfolioListReadResponse {
            return PortfolioListReadResponse(title, count, questions)
        }
    }
}
